### ISSUE TITLE
Initialization models and accuracy@1 in pytorch

### ISSUE THREAD

**Issue by Lovelyczl** [zhang-can/ECO-pytorch#52]
Thanks for your work!
I want to reproduce this paper，But I have some questions about Initialization models and accuracy@1
Q1: does the models use Initialization models in paper if you just set args.pretrained_parts == "both", In other words, weight_url_2d='https://yjxiong.blob.core.windows.net/ssn-models/bninception_rgb_kinetics_init-d4ee618d3399.pth' and models/C3DResNet18_rgb_16F_kinetics_v1.pth.tar
is your Initialization models in your paper ECO-pytorch
Q2: is it normal that baseline is 53.4% when i set args.pretrained_parts == "scratch" and num_segments=16 , could you provide some information about your model accuracy@1 when you set pretrained_parts='scratch', '2D', '3D', 'both','finetune' and segments=16
i am new learner in this, If you can help me, I will be very grateful.
Looking forward to your reply @zhang-can

**Comment by Lovelyczl** [comment]
@chen-ming2019

**Comment by Fovever1** [comment]
Hi czl, Do you run this code successfully?i want to ask you some question.

**Comment by Fovever1** [comment]
i have reproduced this code,  can i talk about this code with you?QQ726421577

**Comment by chen-ming2019** [comment]
@Fovever1  你说的哪个数据集？

**Comment by Fovever1** [comment]
UCF101,我不明白为什么我的是top1=76%,top5=92%,loss=1.0，到第十个epoch的时候，test loss就降不下去了，我是完全按照您的要求来的，为什么结果不一样，我是clone原作者的pytorch代码

**Comment by Fovever1** [comment]
将视频转成帧也是用的TSN的Denseflow提取的，预训练模型用的不是eco_lite_rgb_16F_kinetics_v3.pth，用的是ECO_Lite_rgb_model_Kinetics.pth

**Comment by Fovever1** [comment]
@chen-ming2019

**Comment by chen-ming2019** [comment]
@Fovever1 你这个损失还挺大的，应该是还没有完全收敛，

**Comment by Fovever1** [comment]
@chen-ming2019 您觉得是什么原因造成的，能说下您的具体细节吗

**Comment by chen-ming2019** [comment]
@Fovever1 3好像是新的，应该会好些，我之前用的是2

**Comment by chen-ming2019** [comment]
@Fovever1你最后面全连接层的维度有改吗，他代码上默认的好像是400

**Comment by Fovever1** [comment]
@chen-ming2019 我赶紧看一下

**Comment by Fovever1** [comment]
@chen-ming2019  可是num_class不就是从这传下去的吗？是101，不是400呀，您说的是指的哪个py文件里的，model？吗
![image](https://user-images.githubusercontent.com/31758463/66701102-f701a400-ed2a-11e9-9563-4e6470f7a7fa.png)

**Comment by chen-ming2019** [comment]
@Fovever1对的，是我记混了，那就可能是你参数设置的问题了，这个不太好说

**Comment by Fovever1** [comment]
参数设置我是完全按照opt.py里面来的？您当时还调了其他参数吗？

**Comment by chen-ming2019** [comment]
@Fovever1你把你参数发来，我看看

**Comment by Fovever1** [comment]
因为我要在这个实验上继续优化，发论文才能毕业，现在测试情况不太好，所以希望能得到您的一些帮助，定重谢！您的微信和QQ是多少？

**Comment by Fovever1** [comment]
@chen-ming2019 发我邮箱一下726421577@qq.com

**Comment by chen-ming2019** [comment]
785432809  QQ

**Comment by dandingol03** [comment]
@Fovever1 @chen-ming2019 大佬们，我想请教下，我也是在ufc101上训练，最后top1是75%，请问还要调整那个参数吗,thanks in advance~,我直接按照readme跑的ECO_Lite_rgb_model_Kinetics
